src/test/scala/uk/gov/hmcts/ccd/data/CreateDIVCaseData.scala

```diff
@@ -0,0 +1,154 @@
+package uk.gov.hmcts.ccd.data
+
+import io.gatling.core.Predef._
+import io.gatling.core.feeder.SourceFeederBuilder
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.util.{CcdTokenGenerator, PerformanceTestsConfig}
+
+import scala.concurrent.duration._
+import scala.util.Random
+
+
+object CreateDIVCaseData extends PerformanceTestsConfig {
+
+
+  private val rng: Random = new Random()
```

Comment:
hi level of duplication found here. Searching for `1000000 + rng.nextInt(9999999 - 1000000) + 1` finds 5 instances. This means we could introduce some helper methods
and reuse those rather than duplicating.
Even better we should have all these random values generation code extracted into a dedicated class. And reused everywhere